1.10.10.dev/alpine3.10/trivyignore

```diff
@@ -0,0 +1,3 @@
+# ignore in old airflow versions; it's fixed ONLY in new airflow version
+# lodash: Prototype pollution in utilities function fixed in >=4.17.11
+CVE-2018-16487
```

Comment:
Shouldn't need this anymore -- it should be fixed by one of Ry's commits that we've backported?

Reply:
Removed

Reply:
Looks like Ry's commit didn't resolve it:

```
usr/local/lib/python3.7/site-packages/airflow/www_rbac/yarn.lock
================================================================
Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 1, CRITICAL: 0)

+---------+------------------+----------+-------------------+---------------+--------------------------------+
| LIBRARY | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |             TITLE              |
+---------+------------------+----------+-------------------+---------------+--------------------------------+
| lodash  | CVE-2018-16487   | HIGH     | 3.10.1            | >=4.17.11     | lodash: Prototype pollution in |
|         |                  |          |                   |               | utilities function             |
+---------+------------------+----------+-------------------+---------------+--------------------------------+

Exited with code exit status 1
```

Reply:
Fixed with https://github.com/astronomer/airflow/commit/50acb85652cbe820a0c8668af36a13d7c05e35c3

Reply:
I had to revert the change as it had the following issue when running yarn install:

```
fatal error: too many errors emitted, stopping now [-ferror-limit=]
1 warning and 20 errors generated.
make: *** [Release/obj.target/fse/fsevents.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/Cellar/node/13.8.0/libexec/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:321:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Darwin 19.3.0
gyp ERR! command \"/usr/local/Cellar/node/13.8.0/bin/node\" \"/usr/local/Cellar/node/13.8.0/libexec/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js\" \"build\" \"--fallback-to-build\" \"--module=/Users/kaxilnaik/Documents/Github/astronomer/airflow/airflow/www_rbac/node_modules/fsevents/lib/binding/Release/node-v79-darwin-x64/fse.node\" \"--module_name=fse\" \"--module_path=/Users/kaxilnaik/Documents/Github/astronomer/airflow/airflow/www_rbac/node_modules/fsevents/lib/binding/Release/node-v79-darwin-x64\" \"--napi_version=5\" \"--node_abi_napi=napi\"
gyp ERR! cwd /Users/kaxilnaik/Documents/Github/astronomer/airflow/airflow/www_rbac/node_modules/fsevents
gyp ERR! node -v v13.8.0
gyp ERR! node-gyp -v v5.0.7
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute 'node-gyp build --fallback-to-build --module=/Users/kaxilnaik/Documents/Github/astronomer/airflow/airflow/www_rbac/node_modules/fsevents/lib/binding/Release/node-v79-darwin-x64/fse.node --module_name=fse --module_path=/Users/kaxilnaik/Documents/Github/astronomer/airflow/airflow/www_rbac/node_modules/fsevents/lib/binding/Release/node-v79-darwin-x64 --napi_version=5 --node_abi_napi=napi' (1)
```